src/jit/codegencommon.cpp

```diff
@@ -8644,6 +8644,43 @@ void CodeGen::genFnEpilog(BasicBlock* block)
         unwindStarted = true;
     }
 
+#if !FEATURE_FASTTAILCALL
```

Comment:
If/once `FEATURE_FASTTAILCALL` is enabled for ARM, this will need to change. Would it make sense to change this to something like so that it is ready?

```
#if FEATURE_FASTTAILCALL
if (jmpEpilog && jmpNode->gtOper == GT_JMP)
#else
if (jmpEpilog)
#endif
```

Or maybe not have this under ifdef at all and just do `if (jmpEpilog && jmpNode->gtOper == GT_JMP)` always to keep the code simple.

Reply:
I've updated latest commit with your last suggestion, thank you!